tests/integration/targets/azure_rm_sqlserver/tasks/main.yml

```
@@ -37,13 +64,27 @@
       - output.changed
       - output.state == 'Ready'
 
+- name: Gather facts SQL Server
+  azure_rm_sqlserver_info:
+    resource_group: "{{ resource_group }}"
+    server_name: "sqlsrv{{ random_postfix }}"
+  register: output
+- name: Debug
+  debug:
```

Use `ansible.builtin.debug` or `ansible.legacy.debug` instead
```suggestion
  ansible.builtin.debug:
```

---

I removed the debug tasks